content/help/ops/setup/webhook/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Dynamic Admission Webhooks
+description: Describes Istio's use of Kubernetes webhooks and the related issues that can arise.
+weight: 10
+---
+
+This page assumes you're familiar with the [Kubernetes mutating and
+validating webhook
+mechanisms](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/). Consult
+the Kubernetes API references for detailed documentation of the
+[mutating](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#mutatingwebhookconfiguration-v1beta1-admissionregistration-kubernetes-io)
+and
+[validating](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#validatingwebhookconfiguration-v1beta1-admissionregistration-kubernetes-io)
+webhook configuration.
+
+You may need to modify and/or verify the following specific fields in
```

Comment:
Can you explain why I may need to modify or verify these fields? What the the symptoms that would make me want to do this?